.github/workflows/main.yml

```diff
@@ -60,6 +60,15 @@ jobs:
       - name: Run dotnet format (checks formatting)
         run: make check
 
+      - name: Restore SNK key file
+        run: |
+          echo "$SNK_KEY" | base64 --decode > scylladb.snk
+          sn -p scylladb.snk publickey.snk
+          sn -tp publickey.snk
+        shell: bash
+        env:
+          SNK_KEY: ${{ secrets.SNK_KEY }}
```

Comment:
Question, do you realy need this in here ? what will happen if that file is not present ?